manubot/cite/citeproc.py

```diff
@@ -22,29 +26,29 @@ def csl_item_passthrough(csl_item, set_id=None, prune=True):
     http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html
     https://github.com/citation-style-language/schema/blob/master/csl-data.json
     """
+    csl_item = CSL_Item(csl_item) # wrap dictionary as CSL_Item dictionary
     if set_id is not None:
         csl_item['id'] = set_id
     logging.debug(f"Starting csl_item_passthrough with{'' if prune else 'out'} CSL pruning for id: {csl_item.get('id', 'id not specified')}")
 
     # Correct invalid CSL item types
-    # See https://github.com/CrossRef/rest-api-doc/issues/187
-    if 'type' in csl_item:
-        csl_item['type'] = csl_item_type_fixer.get(csl_item['type'], csl_item['type'])
+    csl_item = csl_item.correct_invalid_type()
 
     if prune:
         # Remove fields that violate the CSL Item JSON Schema
-        csl_item, = remove_jsonschema_errors([csl_item])
-
-    # Default CSL type to entry
-    csl_item['type'] = csl_item.get('type', 'entry')
+        csl_item, = remove_jsonschema_errors([csl_item])    
+        
+    # Default CSL type to 'entry'
+    csl_item = csl_item.set_default_type()
 
     if prune:
         # Confirm that corrected CSL validates
         validator = get_jsonschema_csl_validator()
         validator.validate([csl_item])
+    assert isinstance(csl_item, CSL_Item) # FIXME: Remove if passes
     return csl_item
 
-
+ 
```

Comment:
Maybe also can put a script in repo to run formatter like `autopep8` (`black` or `YAPF`)  with preferred options to sort issues like this quickly? 

Should we use `autopep8 -iaa`? It will change some of the existing formatting though.

Reply:
Whitespace I can remove with `autopep8 -i D:\github\manubot\manubot\cite\citeproc.py --select W291`

Reply:
I do think it would be helpful to enforce black for style. But let's do that in a separate PR after merging all open PRs that are mergeable. Until then, just check the diff using `git diff master` and make sure there aren't these spacing issues. Basically, the PR should not introduce any new formatting issues, but also shouldn't fix unrelated spacing issues in code it doesn't touch.